public/client.js

```diff
@@ -30,4 +30,12 @@ function removeProduct() {
   userRequest.open('post', '/removeProduct');
   userRequest.setRequestHeader("Content-Type", "application/json;charset=UTF-8")
   userRequest.send(JSON.stringify({'product':document.getElementById("removeProduct").value, 'quantity': document.getElementById("quantity").value}));
-}
\ No newline at end of file
+}
+
+/*function searchProduct() {
+  console.log("searching a product...")
+  const userRequest = new XMLHttpRequest();
+  userRequest.open('post', '/db');
+  userRequest.setRequestHeader("Content-Type", "application/json;charset=UTF-8")
+  userRequest.send(JSON.stringify({'product':document.getElementById("addProduct").value}));
+/*
```

Comment:
Minor hiccup here, as comment blocks should be opened with /*, but closed with ***/**. As a result, here we have two opening comment block symbols, so please change the last one on line 41 to */.

Reply:
my bad, i updated it :) Thanks for catching it